service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!-- Order & Related Services -->
+    <service verb="get" noun="FemalePersons" displayName="get the female person details">
+        <in-parameters>
+            <parameter name="gender" default-value="F"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="personList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="personList">
+                <econdition field-name="gender"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="Persons" displayName="get the person details">
+        <in-parameters>
+            <parameter name="maritalStatusEnumId" default-value="MarsMarried"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="personList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="personList">
+                <econdition field-name="maritalStatusEnumId"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="LatestOrders" displayName="get the Latest order">
+        <out-parameters>
+            <parameter name="ordersList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <order-by field-name="placedDate"/>
```

Comment:
Would this order the orderList in ascending or descending?

Reply:
Ascending Order.

Reply:
Updated the recommended changes.
Thank you.